.gitignore

```diff
@@ -15,3 +15,6 @@ gtdb_attribs2.jsonl
 .DS_Store
 .vscode/settings.json
 collections_config.toml
+gtdb_attribs3.jsonl
+gtdb_taxa_count2_rank.jsonl
+gtdb_taxa_count2.jsonl
```

Comment:
I am not aware any tests create those files. Just personal files for testing?

Reply:
yeah. Usually I don't check in .gitignore for this reason but it looks like it got checked in at some point